File: solutions/python/armstrong-numbers/1/armstrong_numbers.py
def is_armstrong_number(number):
    """Takes in a number and assesses whether it is an armstrong number or not
    
    :param number: The number to be evaluated.
    :return: bool - Whether the number is an armstrong number or not
    """

    digits = tuple(str(number))

    suma = 0
    for count, digit in enumerate(digits):
        suma += int(digit) ** len(digits)

    return number == suma

File: solutions/javascript/armstrong-numbers/1/armstrong-numbers.js
export const isArmstrongNumber = (n) => {
    return n === armsCalc(n);
};

function armsCalc(n) {
    let sumOfPowers = 0;
    const digits = String(n).split("");

    for (let digit of digits) {
        sumOfPowers += Math.pow(Number(digit), digits.length);
    }
    
    return sumOfPowers
}
